Src/tim.c

```diff
@@ -52,12 +52,12 @@ void MX_TIM16_Init(void)
   TIM_OC_InitTypeDef sConfigOC;
   TIM_BreakDeadTimeConfigTypeDef sBreakDeadTimeConfig;
 
-  htim16.Instance = TIM16;
-  htim16.Init.Prescaler = 0;
+  htim16.Instance = TIM16; //setting the compare register
+  htim16.Init.Prescaler = 1024; //brings freq down to 46875, need it at 50. So Compare match register 936.5 (+1) to get interrupt freq = 50Hz
```

Comment:
See above comment about CCR registers - I think you have the wrong implementation of this.

Reply:
Definitely did